go.mod

```diff
@@ -0,0 +1,36 @@
+module github.com/openshift/cluster-nfd-operator
+
+require (
+	github.com/NYTimes/gziphandler v1.0.1 // indirect
+	github.com/go-openapi/spec v0.19.0
+	github.com/openshift/api v3.9.0+incompatible
+	github.com/operator-framework/operator-sdk v0.10.1-0.20191003162215-1801b81d5400
+	github.com/spf13/pflag v1.0.3
+	k8s.io/api v0.0.0-20190612125737-db0771252981
+	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/kube-openapi v0.0.0-20190603182131-db7b694dc208
+	sigs.k8s.io/controller-runtime v0.1.12
+	sigs.k8s.io/controller-tools v0.1.10
+	sigs.k8s.io/yaml v1.1.0
+)
+
+// Pinned to kubernetes-1.13.4
```

Comment:
Shouldn't we be writing for k8s 1.16 now?

Reply:
Yep need to update the module. Thanks for the pointer.

Reply:
Hmm,  thinking about it, do we really need the latest kubernetes module? I am not using any features that are not in 1.13 ...

Reply:
Far from being an expert on this, but I try to match this [compatilibity matrix](https://github.com/kubernetes/client-go#compatibility-client-go---kubernetes-clusters) at least for client-go.  Waiting a while for the table update for 1.16 for my operator.

Reply:
Ah good pointer bumped the client-go to release-13.0